docs/_documentations/mdt-che-setupregistries.md

```diff
@@ -1,25 +1,75 @@
 ---
 layout: docs
-title: Installing Codewind on the Cloud
-description: Installing Codewind on the Cloud
-keywords: build, deploy, IBM Cloud Private, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops, OpenShift, OKD
+title: Adding registries in Che
+description: Adding registries in Che
+keywords: build, deploy, install, installing, installation, chart, Helm, develop, cloud, public cloud, services, command line, cli, command, start, stop, update, open, delete, options, operation, devops
 duration: 1 minute
 permalink: mdt-che-setupregistries
 type: document
-order: 20
-parent: root
 ---
 
-### Adding registries in Che
-After Che is started and running, add the container registry that will be used with Codewind.
-- On OpenShift or other Kube platforms, you can push your images to any Docker registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more.
-- On IBM Cloud Private, you can push your image to any Docker registry *except* the internal Docker registry.
+# Adding registries in Che
+After Che is started and running, add the container registry to be used with Codewind.
+1. From the Codewind Project Explorer view, right-click **Projects** and select **Image Registry Manager**.
+2. From the **Image Registry Manager**, click **Add New**.
+3. On OpenShift and other Kubernetes platforms, push your images to a container registry, such as Docker Hub, Quay.io, Google Cloud Registry (GCR), and more. Enter the following information to add the registries:
+  - Registry server name or domain name: `<registry-to-push-images-to>`
+  - User name: `<Your username>`
+  - Password or API key: `<Your password or API key>`
+4. Next, enter the namespace that you want to use to push your registry to. After you enter all the information, from the **Image Registry Manager**, you can see values for the **Address**, **Username**, and **Namespace** of your latest registry.
```

Comment:
Should be something like: `If you're using that registry as a push registry, enter the namespace that you wish to push your images to`

I think we should also add a note somewhere about the examples we have down below.